docs/updating/updating_to_3.3.md

```diff
@@ -7,6 +7,16 @@ If you are updating from an earlier version, start with updating your installati
 
 First, create an upgrade branch in git and commit your work.
 
+If you have not done it before, add the relevant meta-repository as `upstream` remote:
+
+``` bash
```

Comment:
I'd put the "or"s outside of code blocks. I can imagine someone copy-pasting the whole block and fetting back yo us.
Even better, is it a whole lot of work to put three code blocks into separate tabs, as with installation options for different editions?

Otherwise, I approve